external/nghttp2/Kconfig

```diff
@@ -0,0 +1,12 @@
+config ENABLE_NGHTTP2
+	bool "Enable NGHTTP2"
+	default n
+	depends on MBEDTLS
+	---help---
+		nghttp2 is an implementation of HTTP/2 and its header compression algorithm HPACK.
+
+if ENABLE_NGHTTP2
+config NGHTTP2_DEBUG
+	bool "Nghttp2 Debug"
+	default n
```

Comment:
How about make a dependancy with DEBUG?
If it has, when we disable DEBUG, all of debug feature will be disabled automatically. User does not need to check all of debug feature at each module.
```
depends on DEBUG
```

Reply:
@sunghan-chang  ok, made changes.
But with 'default n' i dont think this issue would arise

Reply:
That is a default value. When config does not have any value, default value will be set. But I concern the case that nghttp2 debug is set to y. If we keep it as of now, we should turn off debug and nghttp2 debug separately. But if we make a dependancy, just turnning off debug is enough to disable all of debug feature.

Reply:
@sunghan-chang Yes, true. Made changes